Jenkinsfile

```diff
@@ -20,35 +20,96 @@ pipeline {
                 parallel(
                     linux: {
                         node('linux-large') {
+                            echo sh(returnStdout: true, script: 'env')
                             timeout(time: 3, unit: 'HOURS') {
                                 checkout scm
                                 withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
-                                    sh 'mvn clean install -pl !distribution/test/itests/test-itests-ddf'
-                                    sh 'mvn install -pl distribution/test/itests/test-itests-ddf -nsu'
+                                    sh 'mvn clean install -B -q -T 1C -pl !distribution/test/itests/test-itests-ddf'
+                                    sh 'mvn install -B -Dmaven.test.redirectTestOutputToFile=true -pl distribution/test/itests/test-itests-ddf -nsu'
                                 }
                             }
                         }
-                    }, windows: {
-                        node('proxmox-windows'){
+                    },
+                    windows: {
+                        node('proxmox-windows') {
                             timeout(time: 3, unit: 'HOURS') {
                                 checkout scm
                                 withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
-                                    bat 'mvn clean install -pl !distribution/test/itests/test-itests-ddf'
-                                    bat 'mvn install -pl distribution/test/itests/test-itests-ddf -nsu'
+                                    bat 'mvn clean install -B -q -T 1C -pl !distribution/test/itests/test-itests-ddf'
+                                    bat 'mvn install -B -Dmaven.test.redirectTestOutputToFile=true -pl distribution/test/itests/test-itests-ddf -nsu'
                                 }
                             }
                         }
                     }
                 )
             }
         }
+        stage('Static Analysis') {
+            steps {
+                parallel(
+                        owasp: {
+                            node('linux-large') {
+                                checkout scm
+                                withMaven(maven: 'M3', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings') {
+                                    sh 'mvn install -q -B -Powasp -DskipTests=true -Dpmd.skip=true -Dfindbugs.skip=true -Djacoco.skip=true -Dcheckstyle.skip=true -pl !libs/owasp-diff-runner,!distribution/docs'
```

Comment:
❓ Can't we use `-DskipStatic=true` here instead of listing every single static analysis tool we want to skip?